service/service.go

```
@@ -16,10 +19,26 @@ func (s Service) HealthCheck(ctx context.Context, empty *emptypb.Empty) (*api.He
 }
 
 func (s Service) GetTicks(ctx context.Context, request *api.GetTicksRequest) (*api.GetTicksResponse, error) {
-	//TODO implement me
-	panic("implement me")
+	tick, err := s.db.GetTicks(ctx, request.Symbol)
+	if err != nil {
```

golangci-lint (1.62.2)

SA9003: empty branch

(staticcheck)